static/app/components/smartSearchBar/index.tsx

```diff
@@ -1009,10 +1009,10 @@ class SmartSearchBar extends React.Component<Props, State> {
     }
 
     if (cursorToken.type === Token.FreeText) {
-      const autocompleteGroups = [
-        await this.generateTagAutocompleteGroup(cursorToken.text),
-      ];
-      this.setState({searchTerm: cursorToken.text});
+      const lastToken = cursorToken.text.trim().split(' ').pop() ?? '';
```

Comment:
because we always take the last token it might get a little weird for things like `free text browser` when the user moves the cursor over to `text` autocomplete entries will still be shown for `browser` because we don't have any cursor position logic in here. Not sure if that is worth adding, but it seems a little hacky to add that in :/

Reply:
I think it's OK